### ISSUE TITLE
Help text in `plotcoeffs` and `plotregion`

### ISSUE THREAD

**Issue by trefethen** [chebfun/chebfun#1521]
In connection with #1518 I've just looked at the help text for `plotcoeffs`, which presumably will be a model for the help text for the new `plotregion`.  I would like to suggest a change to make this text meaningful for users.

What does `plotcoeffs` plot?  Currently users are told the following:
```
What 'coefficients' means in this context is dictated by the 'tech' that is
being used to represent F. See the plotcoeffs method at the relevant tech
level for more details. Note that you can find what tech is being used by
calling GET(F, 'tech').
```
I see two related issues with this.  First, it is obscure and doesn't tell people that we're talking about Chebyshev and Fourier coefficients.  Second, it talks about techs, which are not supposed to appear at user level.  Basically this text seems aimed at developers rather than users.

I suggest changing this paragraph to something like what's below.  @aaustin141, is this something you would be comfortable with doing?  
```
As of Chebfun version 5, the coefficients plotted by `plotcoeffs` are either
Chebyshev coefficients for Chebyshev-based chebfuns or Fourier coefficients
for periodic chebfuns ("trigfuns"). 
```
As a small additional correction to the help text, I think the opening sentence
```
plotcoeffs(F) plots the coefficients...
```
should be changed to
```
plotcoeffs(F) plots the absolute values of the coefficients...
```

**Comment by kuanxu** [comment]
Should we also remove the words "If S contains a string 'NOEPSLEVEL', the epslevel is not plotted."? Anyhow, we are not plotting `epslevel` now. But if people think that we should do that after some months have elapsed, I'm totally OK.

**Comment by trefethen** [comment]
Yes, we should certainly remove those words!  Thanks for noticing.

**Comment by aaustin141** [comment]
Closed in #1524.
